src/eth/rpc.ts

```diff
@@ -28,12 +29,15 @@ export const getDelegators = async ():Promise<Array<string>> => {
           let batch = (await fetchSubgraph({
             query: `{
               delegators(skip: ${delegators.length}, where:{ bondedAmount_not: 0 }) {
-                id
+                id,
+                delegate {
+                    id
+                }
               }
             }`,
           })).data.delegators
     
-          batch = await Promise.all(batch.map(d => getEarnings(d.id, snapshotRound)))
+          batch = await Promise.all(batch.map(d => getEarnings(d.id, snapshotRound, d.delegate.id)))
```

Comment:
```suggestion
          batch = await Promise.all(batch.map(d => getDelegatorSnapshot(d.id, d.delegate.id, snapshotRound)))
```

with `getDelegatorSnapshot` defined as:

```typescript
async function getDelegatorSnapshot(
    delegator: string,
    delegate: string,
    snapshotRound: BigNumber
): Promise<{delegator: string, delegate: string, pendingStake: BigNumber, pendingFees: BigNumber> {
    const earnings = await getEarnings(delegator, snapshotRound)
    return {
        ...earnings,
        delegate,
    }
}
```